src/pointers.js

```diff
@@ -0,0 +1,10 @@
+import pointer from "./pointer.js";
+import sourceEvent from "./sourceEvent.js";
+
+export default function(events, node) {
+  if (events.target) { // i.e., instanceof Event, not TouchList or iterable
```

Comment:
I think I’d prefer `events instanceof Event` here, but our current custom events in d3-drag *etc.* don’t extend Event, so that isn’t possible. It might be possible to change them to extend Event, although for all I know, that may introduce other problems.